node/src/main.rs

```diff
@@ -51,6 +51,13 @@ lazy_static! {
         .map(|s| usize::from_str(&s)
              .unwrap_or_else(|_| panic!("failed to parse env var GRAPH_TOKIO_THREAD_COUNT")))
         .unwrap_or(100);
+
+    static ref BLOCK_CACHE_CLEANUP_FREQ: Duration = env::var("GRAPH_BLOCK_CACHE_CLEANUP_FREQ")
+        .ok()
+        .map(|s| u64::from_str(&s)
+             .unwrap_or_else(|_| panic!("failed to parse env var GRAPH_BLOCK_CACHE_CLEANUP_FREQ")))
+        .map(|secs| Duration::from_secs(secs))
+        .unwrap_or(Duration::from_secs(0));
```

Comment:
Can we move this into `BlockIngestor`, rename it to `GRAPH_ETHEREUM_BLOCK_CLEANUP_INTERVAL`, and also drop the frequency argument to `into_polling_stream()`? I feel this is internal to Ethereum, for other chains in the future we may want to configure different intervals.

Reply:
I am actually thinking that it shouldn't be an interval at all, and we should do that cleanup after each block we ingest, i.e., make that environment variable a boolean. That way, we will most often clean up just a small number of blocks, and it'll help in keeping `ethereum_blocks` small.

Reply:
I made it a boolean, and changed the name to `GRAPH_ETHEREUM_CLEANUP_BLOCKS` and moved it into the block ingestor